# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/encoding/textdecoder-fatal-single-byte.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/encoding/textdecoder-fatal-single-byte.any.js.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/encoding/textdecoder-fatal-single-byte.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -0,0 +1,33 @@
+[textdecoder-fatal-single-byte.any.worker.html?6001-7000]
+
+[textdecoder-fatal-single-byte.any.html?3001-4000]
+
+[textdecoder-fatal-single-byte.any.html?5001-6000]
+
+[textdecoder-fatal-single-byte.any.worker.html?5001-6000]
+
+[textdecoder-fatal-single-byte.any.worker.html?7001-last]
+
+[textdecoder-fatal-single-byte.any.worker.html?1001-2000]
+
+[textdecoder-fatal-single-byte.any.html?6001-7000]
+
+[textdecoder-fatal-single-byte.any.html?1001-2000]
+
+[textdecoder-fatal-single-byte.any.html?2001-3000]
+
+[textdecoder-fatal-single-byte.any.worker.html?1-1000]
+  expected:
+    if (os == "linux") and not webrender and not debug and (processor == "x86_64"): ["OK", "TIMEOUT"]
+
+[textdecoder-fatal-single-byte.any.worker.html?4001-5000]
+
+[textdecoder-fatal-single-byte.any.html?1-1000]
+
+[textdecoder-fatal-single-byte.any.worker.html?2001-3000]
+
+[textdecoder-fatal-single-byte.any.worker.html?3001-4000]
+
+[textdecoder-fatal-single-byte.any.html?4001-5000]
+
+[textdecoder-fatal-single-byte.any.html?7001-last]